### PR TITLE
History Map - Delete logic

### PR DESCRIPTION
**Pull request by HlibBondarev**
Done:

1) Created DTO: DeleteStreetcodeToponymRequestDto and DeleteStreetcodeToponymResponseDto; 
2) Created Mediator Handler: DeleteStreetcodeToponymHandler, DeleteStreetcodeToponymCommand and DeleteStreetcodeToponymRequestDtoValidator; 
3) Created DeleteStreetcodeToponymHandlerTests; 
4) Created DeleteStreetcodeToponymRequestDtoValidatorTests; 
5) Added Delete action to StreetcodeToponymController

### CODE REVIEW

**Review by yaninavoda — APPROVED** [review]
Everything works fine)

File: Streetcode/Streetcode.WebApi/Controllers/Toponyms/StreetcodeToponymController.cs [hunk 1]
```diff
@@ -1,11 +1,18 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using Streetcode.BLL.DTO.StreetcodeToponym;
+using Streetcode.BLL.MediatR.StreetcodeToponym.Delete;
 using Streetcode.BLL.MediatR.StreetcodeToponym.Create;
 
 namespace Streetcode.WebApi.Controllers.Toponyms;
 
 public class StreetcodeToponymController : BaseApiController
 {
+    [HttpDelete]
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.WebApi/Controllers/Toponyms/StreetcodeToponymController.cs:10]: Replace. [HttpDelete] under [HttpPost]

**Review by OlhaPivniuk — APPROVED** [review]
Replace please. [HttpDelete] under [HttpPost]

**Comment by HlibBondarev** [comment]
> Replace please. [HttpDelete] under [HttpPost]

It's done

File: Streetcode/Streetcode.WebApi/Controllers/Toponyms/StreetcodeToponymController.cs (hunk 1 above)
Comment (HlibBondarev) [reply Streetcode/Streetcode.WebApi/Controllers/Toponyms/StreetcodeToponymController.cs]: It]s done

**Pull request merged**